### PR TITLE
[Estuary][PVR] Recordings Info dialog: Fix 'Play recording' button.

### PR DESCRIPTION
**Pull request by ksooo**
Reported here: https://github.com/xbmc/xbmc/pull/10333#issuecomment-242981005

Broken after #10333 due to 'interesting' implementation of the 'play recording' button in Estuary. ;-)

@axbmcuser fyi
@Jalle19 mind doing the pvr code review?
@BigNoid for the review of the Estuary change?

### CODE REVIEW

File: addons/skin.estuary/1080i/DialogPVRInfo.xml [hunk 1]
```diff
@@ -90,11 +90,9 @@
 					<param name="visible" value="Window.IsActive(PVRGuideInfo)" />
 				</include>
 				<include content="InfoDialogButton">
-					<param name="id" value="5010" />
+					<param name="id" value="8" />
```
Comment (BigNoid) [inline addons/skin.estuary/1080i/DialogPVRInfo.xml]: There's already button id ```8``` on ln 83. Better remove this one and make that one visible on ```Window.IsActive(PVRGuideInfo) | Window.IsActive(PVRRecordingInfo)```

Comment (ksooo) [inline addons/skin.estuary/1080i/DialogPVRInfo.xml]: Okay, will do.

Comment (ksooo) [inline addons/skin.estuary/1080i/DialogPVRInfo.xml]: @BigNoid the labels of the two 8 buttons are different! How to handle this?

Comment (BigNoid) [inline addons/skin.estuary/1080i/DialogPVRInfo.xml]: They both have ```$LOCALIZE[19687]``` as label?

Comment (ksooo) [inline addons/skin.estuary/1080i/DialogPVRInfo.xml]: Oooops, too early in the morning. All good, the labels are the same. :-)

File: xbmc/pvr/windows/GUIWindowPVRBase.h [hunk 2]
```diff
@@ -109,13 +111,12 @@ namespace PVR
...
     virtual bool ActionInputChannelNumber(int input);
 
-    virtual bool PlayRecording(CFileItem *item, bool bPlayMinimized = false, bool bCheckResume = true);
     virtual bool PlayFile(CFileItem *item, bool bPlayMinimized = false, bool bCheckResume = true);
     virtual void ShowEPGInfo(CFileItem *item);
     virtual void ShowRecordingInfo(CFileItem *item);
     virtual bool UpdateEpgForChannel(CFileItem *item);
     virtual void UpdateSelectedItemPath();
-    bool CheckResumeRecording(CFileItem *item);
+    static bool CheckResumeRecording(CFileItem *item);
```
Comment (Jalle19) [inline xbmc/pvr/windows/GUIWindowPVRBase.h]: Unrelated?

Comment (ksooo) [inline xbmc/pvr/windows/GUIWindowPVRBase.h]: > Unrelated?

No, PlayRecording, which is now static, calls CheckResumeRecording, thus it needs to be static, too. ;-)

**Comment by ksooo** [comment]
@BigNoid addressed your comment re the Estuary change.

File: xbmc/pvr/windows/GUIWindowPVRBase.h (hunk 2 above)
Comment (Jalle19) [inline xbmc/pvr/windows/GUIWindowPVRBase.h]: Thanks, was hard to tell from the diff.

**Comment by ksooo** [comment]
jenkins build this please

**Comment by axbmcuser** [comment]
Fix works for me. Tested with setting "Show information" (as before) and "Choose" (selecting play, resume and information afterwards). Thanks!

**Comment by ksooo** [comment]
Travis error is unrelated
